scripting.md

```diff
@@ -419,6 +419,25 @@ wichever other policy.
 
 See [Continuous Integration](testing.html#continuous-integration).
 
+# Bootstrapping an app
+
+## With SBCL
+
+To automate the process of installing quicklisp (after downloading `quicklisp.lisp`):
+~~~
+$ sbcl --non-interactive \
+       --load quicklisp.lisp \
+       --eval '(quicklisp-quickstart:install)' \
+       --eval '(ql::without-prompting (ql:add-to-init-file))'
+~~~
+
+You can also automate the process of installing dependencies, compiling your system, and running your app like so:
+~~~
+$ sbcl --eval '(ql:quickload :my-app)' \
+       --eval '(asdf:load-system :my-app)' \
+       --eval '(my-app:main)'
+~~~
```

Comment:
Seems like this would have its place better on scripting (https://lispcookbook.github.io/cl-cookbook/scripting.html).

There, we mention other useful stuff (the `--no-interactive` option so that the debugger doesn't pop up), how to parse CLI args, etc.

Reply:
You mean in a different section in scripting?